cloudflare/provider.go

```diff
@@ -116,6 +116,7 @@ func Provider() *schema.Provider {
 			"cloudflare_zones":                       dataSourceCloudflareZones(),
 			"cloudflare_zone":                        dataSourceCloudflareZone(),
 			"cloudflare_zone_dnssec":                 dataSourceCloudflareZoneDNSSEC(),
+			"cloudflare_access_identity_provider":    dataSourceCloudflareAccessIdentityProvider(),
```

Comment:
same as https://github.com/cloudflare/terraform-provider-cloudflare/pull/1300/files#r743364820, we'll sort this alphabetically please.